src/hooks/mod.rs

```diff
@@ -54,7 +54,7 @@ pub struct AuthContext<'c> {
     /// The database name, if applicable.
     pub database_name: Option<&'c str>,
 
-    /// The inner-most trigger or view responsible for the access attempt.
+    /// The innermost trigger or view responsible for the access attempt.
```

Comment:
https://sqlite.org/search?s=d&q=inner-most

Reply:
Are you implying that SQLite docs should be updated as well, or that they introduced a new keyword with a new meaning?  https://www.merriam-webster.com/dictionary/innermost -- is this the meaning?